doc/dependencies.md

```diff
@@ -5,7 +5,7 @@ These are the dependencies currently used by Bitcoin Core. You can find instruct
 
 | Dependency | Version used | Minimum required | CVEs | Shared | [Bundled Qt library](https://doc.qt.io/qt-5/configure-options.html#third-party-libraries) |
 | --- | --- | --- | --- | --- | --- |
-| Berkeley DB | [4.8.30](https://www.oracle.com/technetwork/database/database-technologies/berkeleydb/downloads/index.html) | 4.8.x | No |  |  |
+| Berkeley DB | [4.8.30](https://www.oracle.com/technetwork/database/database-technologies/berkeleydb/downloads/index.html) | 4.7.x | No |  |  |
```

Comment:
Any reason to lower the minimum? I am not aware of a single user or even operating system that provides this version. The only versions that we have been testing in the past are 4.8, 5.1 and 5.3

Reply:
> I am not aware of a single user or even operating system that provides this version.

You mean you don't want to retain compatibility with Bitcoin Core releases before 0.4.0 (when the BDB dependency was changed from 4.7 to 4.8).

Reply:
It was done just for completeness since 4.7 does technically work.

Reply:
I've set it back to 4.8 as the minimum.

Reply:
Thanks, that seems safer given that Travis only lets of test new versions of BDB against the 4.8 hardcode in previous releases, and I don't see us adding 0.4.0 to the previous release test suite (though that would be cool).